config.public.json

```
@@ -215,6 +215,10 @@
         "6.topic: steam": [
             "pkgs/games/steam"
         ],
+	"6.topic: systemd": [
+            "pkgs/os-specific/linux/systemd",
+            "nixos/modules/system/boot/systemd"
+	],
```

Alignment's slightly off.

```suggestion
		"6.topic: systemd": [
            "pkgs/os-specific/linux/systemd",
            "nixos/modules/system/boot/systemd"
		],
```

---

Also please do not mix tabs and spaces.